src/node.rs

```diff
@@ -839,9 +839,46 @@ impl EthNamespaceT for InMemoryNode {
 
     fn get_transaction_by_hash(
```

Comment:
Looks good to me 👍 . Can we please move this function up above Line `755` now that it's implemented? (we have a split in this file above and below the `// Methods below are not currently implemented.` comment 😄 )